src/test/java/io/cdap/plugin/add/field/MultiFieldAdderConfigTest.java

```diff
@@ -32,6 +34,23 @@
   private static final MultiFieldAdderConfig VALID_CONFIG = new MultiFieldAdderConfig(
     "test_1:s_value_1,test_2:s_value_2,test_3:s_value_3"
   );
+  private static final String ALL_KEYS_MACRO = "${test_1}:value_1,${test_2}:value_2,${test_3}:value_3";
+  private static final String NONE_KEY_MACRO = "test_1:${value_1},test_2:value_2,test_3:${value_3}";
+  private static final String NESTED_MACRO_KEY = "${test_1${nested_1}},value_1,${test_2${nested_2}}:value_2," +
+          "${test_3${nested_3}}:value_3";
+  private static final String NESTED_MACRO_VALUE = "test_1:${value_1${nested_1}},test_2:value_2," +
+          "test_3:${value_3${nested_3}}";
+  private static final String PARTIAL_MACRO_KEYS = "test_1:${value_1},test_2:value_2,test_3:${value_3}" +
+          ",${test_4}:value_4,${test_5}:value_5";
+  private static final String SECURE_MACRO = "test_1:${value_1},test_2:value_2,test_3:${value_3}," +
```

Comment:
please also add a case where whole config is macro (to make sure backwards compatibility) is maintained

Reply:
PR Updated.